pkg/nspredicate/nspredicate.go

```diff
@@ -0,0 +1,48 @@
+package nspredicate
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"sigs.k8s.io/controller-runtime/pkg/event"
+)
+
+// NewNamespacePredicateFromEnvVar looks up the provided environment
+// variable, parses the value, and returns a NamespacePredicate
+func NewFromEnvVar(envVar string) (NamespacePredicate, error) {
+	namespaces, found := os.LookupEnv(envVar)
+	if !found {
+		return NamespacePredicate{}, fmt.Errorf("Environment variable %s not found", envVar)
```

Comment:
It is working with operator-sdk local and unit tests? 
This project had a unit test added and coveralls which is not appearing now in this PR as the CI validation. Do you know why?